### PR TITLE
Update Skyline_Logical_Layer.md

### PR DESCRIPTION
**Pull request by ConanVanHove**
added more help info

### CODE REVIEW

File: connector/doc/Skyline_Logical_Layer.md [hunk 1]
```diff
@@ -38,23 +38,256 @@ This connector uses a virtual connection and does not require any input during e
...
+- The **Conditions** page displays a table with the defined conditions and their result (*True* or *False*) and settings.
+- The **Monitors** page displays three tables for the parameter value, element alarm, and view alarm monitors.
+- The **User-Defined Variables** page displays a table with the defined static variables.
+  
+The connector supports **importing** CSV files in order to easily provision monitors and conditions.
 
-The connector supports **importing** CSV files that have been exported with the built-in DataMiner table export function. For the **Parameter Value Monitors** table, the import will be based on the element name that was filled in. The import will fail if any of the entries is not correct. The file has to be available from the server, e.g. in the DataMiner Documents folder.
+### Adding/Editing/Deleting a Parameter Monitor
+
+In this table, you can define monitors that can subscribe to a single parameter. This can be either a standalone parameter or a specific index of a table parameter.
```
Comment (MariekeGO) [inline connector/doc/Skyline_Logical_Layer.md:55]: In which table do you mean? Could you make sure this is clear?
This comment also applies for line 106 and 165.

File: connector/doc/Skyline_Logical_Layer.md [hunk 2]
```diff
@@ -38,23 +38,256 @@ This connector uses a virtual connection and does not require any input during e
...
+
+   The condition will be added, and the result will be updated to *True* or *False*.
+
+> [!NOTE]
+>
+> - The condition will only be added if the condition name you defined does not exist yet.
+> - No validation is done on the condition definition, and it is added as you defined it. The logic will only try to resolve it taking into account the expected correct syntax and the configured monitors and static variables. If all is well, the logic will resolve the condition to *True* or *False*. Otherwise, it will result in *Unknown* or even possibly *Infinite Loop*. If this happens, you have most likely made a syntax error or a typo in one of the monitor names or compare values.
+
+Explain Visualize
+Explain Corrective Action Text - Automatic Correction - Corrective Action Script - Owner - Timestamp
```
Comment (MariekeGO) [inline connector/doc/Skyline_Logical_Layer.md:274]: What is the purpose of this text? could you clarify this?

File: connector/doc/Skyline_Logical_Layer.md [hunk 3]
```diff
@@ -38,23 +38,256 @@ This connector uses a virtual connection and does not require any input during e
...
+   Use the Shift or Ctrl keys in case you want to select several static variables.
+
+1. Right-click your selection and select **Delete selected item(s)** in the context menu.
+
+1. In the confirmation box, click **OK** if you are sure you want to delete the selected items.
+
+> [!NOTE]
+> Only static variables that are not used in any condition can be deleted. If you try to delete static variables, and one or more of the selected static variables is used in a condition, none of the selected static variables will be deleted.
+
+### Adding/Editing/Deleting Conditions
```
Comment (MariekeGO) [inline connector/doc/Skyline_Logical_Layer.md:248]: Info on editing/deleting conditions is still missing in this section. Could you add this?

**Review by MariekeGO — CHANGES_REQUESTED** [review]

File: connector/doc/Skyline_Logical_Layer.md (hunk 1 above)
Comment (ConanVanHove) [reply connector/doc/Skyline_Logical_Layer.md:55]: Yes, I will make this clear. I will review the whole document with this in mind and resolve any potential gaps.

File: connector/doc/Skyline_Logical_Layer.md (hunk 3 above)
Comment (ConanVanHove) [reply connector/doc/Skyline_Logical_Layer.md:248]: Indeed, overlooked that one. Nice catch! Will add it.

File: connector/doc/Skyline_Logical_Layer.md (hunk 1 above)
Comment (ConanVanHove) [reply connector/doc/Skyline_Logical_Layer.md:55]: This should make it clear for line 56:
### Adding/Editing/Deleting a Parameter Monitor to/of/from the Parameter Value Monitors table

And this for line 106:
### Adding/Editing/Deleting Conditions to/of/from the Conditions table

Unclear for line 165 as I don't see any reference there to a table.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**